.bem/templates/js.js

```diff
@@ -14,16 +14,17 @@ module.exports = function (entity, naming) {
         "import { decl } from 'bem-react-core';",
```

Comment:
I can add it as well but I think you'll need one more iteration to introduce proper templates for other techs anyways?